security/src/test/java/pm/gnosis/heimdall/security/impls/AesEncryptionManagerTest.kt

```diff
@@ -26,7 +26,7 @@ class AesEncryptionManagerTest {
     private lateinit var application: Application
 
     @Mock
-    private lateinit var fingerprintManagerMock: CompatSafeFingerprintManager
+    private lateinit var fingerprintHelperMock: DefaultFingerprintHelper
```

Comment:
you should probably use the `FingerprintHelper` interface here instead of the concrete implementation